lib/controller.js

```diff
@@ -52,6 +52,10 @@ Tessel.list = function(opts) {
 
   return new Promise(function(resolve, reject) {
 
+    if (opts.verbose === false) {
```

Comment:
It's useful to disable all the logging from the test rig. Other API consumer may also appreciate this.